client/lib/analytics/track-resurrections/index.jsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect } from '@wordpress/element';
```

Comment:
We could use that directly from `react`.

Reply:
This has been updated to reflect the requested change.